tg_bot/modules/welcome.py

```diff
@@ -140,6 +142,23 @@ def new_member(bot: Bot, update: Update):
             if sent:
                 sql.set_clean_welcome(chat.id, sent.message_id)
 
+        mute_user = sql.get_mute_pref(chat.id)
+        if mute_user:
+            try:
+                bot.restrict_chat_member(chat.id, new_mem.id,
```

Comment:
So, you mute them permanently? You don't give an unmute button? I don't believe it should be a hard limit, especially since there's no batch unmute option (and there shouldn't be)